src/clients/scc_auto.rb

```diff
@@ -212,6 +223,35 @@ def write
       true
     end
 
+    # Select the product from the control.xml (for the online installation medium)
+    # @return [Boolean] true on success, false on failure
+    def online_medium_config
+      # import the GPG keys before refreshing the repositories
+      Yast::Packages.ImportGPGKeys
+
+      products = Y2Packager::ProductControlProduct.products
+      ay_product = Profile.current.fetch("software", {}).fetch("products", []).first
+
+      if !ay_product
+        # TRANSLATORS: error message, %s is the XML path, e.g. "software/products"
+        Report.Error(_("Missing product specification in the %s section") % "software/products")
+        return false
+      end
+
+      control_product = products.find { |p| p.name == ay_product }
+
+      if !control_product
+        # TRANSLATORS: error message, %s is a product ID, e.g. "SLES"
+        Report.Error(_("Product %s not found") % ay_product)
```

Comment:
and here issue with invalid value.

Reply:
:question: :question: :question: Um, what exactly?

Reply:
I just mean different type of autoyast issue https://github.com/yast/yast-autoinstallation/blob/master/src/lib/autoinstall/autoinst_issues/invalid_value.rb

but if it is not yet so ready, feel free to keep it as it is.